### DISCUSSION TITLE
Mirth capabilities

### DISCUSSION THREAD

**Discussion by jharris110478** [nextgenhealthcare/connect#5438] [Q&A]
Can Mirth insert a JSON with a Base64 PDF back into an existing HL7 result?

**Comment by pacmano1** [comment]
Yes.  But your question is not clear.

Do you mean "I have a base64 string in a field in a JSON blob, can I take that base64 string and put it into an Hl7 field?"    That answer to that is yes.   It is a simple assignment.   The fact it is base64 is not really all the relevant, it's just a string.  Although you need to be sensitive to CRLFs.   There are a couple of YouTube videos that show HL7 to JSON.  For example: https://www.youtube.com/watch?v=pn_AlHfgqVI

**Reply by jharris110478** [reply]
Yes , it is a Base64 coming back to us in a JSON payload via GET.  What I am trying to do is insert it back into the existing HL7 message so that it goes back into the LIS system .

**Reply by pacmano1** [reply]
Yes, that is possible.   What have you tried?  Post your code attempt.  Reminder to use code formatting when you post code here.
